### ISSUE TITLE
Dark Theme

### ISSUE THREAD

**Issue by hakluke** [ehrishirajsharma/SwiftnessX#14]
The title says it all. I'd love to see a dark theme on this app.

**Comment by ehrishirajsharma** [comment]
Yes, we are working on this, you can check here: https://github.com/ehrishirajsharma/SwiftnessX/wiki/Upcomings

I will keep this open for better tracking

**Comment by hakluke** [comment]
That's awesome news!

**Comment by GouveaHeitor** [comment]
Any update?

**Comment by ehrishirajsharma** [comment]
It is planned to be released in **v0.3.** We are basically going to use similar approach of VSCode with `config.json` file which will allow users to change storage (to support Drive and Dropbox) & backups path and tweak elements' colours , e.g.

```
// Backup Frequency is calculated on minutes basis.
// Backup also preserves a daily copy by default.

{
    "StorageConfig": [

        { 
        "backupFrequencey": "30",
        "BackupAmount": "10",
        "StorageFilePath": "/Users/rishirajsharma/Dropbox-dir/",
        "BackupStoragePath": "/Users/rishirajsharma/Documents/Swiftness/Backups"                      
         }
                    ],
                    
    "CustomTheme":[
    {
    "LeftPanel":"#74503D",
    "EditorBackground":"#74503D",
    "TextColour":"#000",
    "Symbols":"#000",
    "CodeBlock": "#987234",
    "SeverityColours": ["#12324", "#55832" , "#12499", "D4231B", "94321G"]
            }    
       ]
     }
``` 

By default we are going to set it with this dark theme:

![image](https://user-images.githubusercontent.com/35542790/58954805-cb3c4380-87b7-11e9-9394-22087c255a18.png)

**Comment by hakluke** [comment]
Looks great @ehrishirajsharma

**Comment by sleeyax** [comment]
Am I missing something or are these upcoming features delayed? Still no `v0.3` after more than a few months. I would love a dark theme too.

**Comment by Myk4my** [comment]
PLEASE MAKE DARK THEME AVAILABLE FOR US NOWWWWW!!!

**Comment by n0kovo** [comment]
> Am I missing something or are these upcoming features delayed? Still no `v0.3` after more than a few months. I would love a dark theme too.

Make that 5 years.

**Comment by sleeyax** [comment]
> > Am I missing something or are these upcoming features delayed? Still no `v0.3` after more than a few months. I would love a dark theme too.
> 
> Make that 5 years.

It's amusing to revisit this thread after such a long time. I had to double-check what we were discussing, as I don't quite recall this application. These days, I use Obsidian for general markdown documents and Logseq for notes and to-do lists.
